make_dataset.py

```diff
@@ -220,51 +336,166 @@ def parse_args(args_input=None):
                         csv_path=csv_path,
                         read_note_csv_kwargs=read_note_csv_kwargs)
                     for csv_path in tqdm(csv_paths, desc="Cleaning csv data")]
+    np.random.shuffle(compositions) # This is important for join_notes
```

Comment:
Why is this important?

Reply:
Actually, not specifically for join_notes any more, but it is still important generally.

With the same degradation proportions, 2 runs will perform the degradations in exactly the same order where possible (i.e., first composition: join_notes, 2nd: split_notes, ....). So, if we use the same datasets without shuffling, the same degradation would be performed on the same excerpt every time.